rds-deployment-HA/azuredeploy.json

```diff
@@ -0,0 +1,397 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+      }
+    },
+    "loadBalancer": {
```

Comment:
This is an existing loadbalancer which is deployed through RDS basic template. Information regarding this is already published in README file. The idea of building this resource is identical to primary deployment. The loadbalancer is defined in this template because the VM creation in this template is dependent on loadbalancer.